db/seeds/dev/01_classes.js

```diff
@@ -0,0 +1,15 @@
+const classesData = require('../../../data/classes');
+
+exports.seed = async (knex) => {
+  try {
+    await knex('classes').del()
+
+    let classesPromises = classesData.map(item => {
+      return knex('classes').insert(item);
+    });
+
+    return Promise.all(classesPromises);
```

Comment:
Going through iteration every element in classes data is added to 'classes' table in DB.